src/components/Image.js

```diff
@@ -33,7 +33,7 @@ export const Image = ({
   borderRadius,
   refreshInterval
 }) => {
-  const [uri, setUri] = useState(null);
+  const [source, setSource] = useState(null);
```

Comment:
i have problems to think over this completely, but would setting `sourceProp` as initial value here make sense? but i guess there are cases, where this does not apply because of the later condition in the effect, right?